tutorials/R/spatial-data/spatialData.Rmd

```diff
@@ -22,331 +23,514 @@ connecting remote sensing with ground-based measurements, the methods in
 the <a href="https://www.neonscience.org/tree-heights-veg-structure-chm" target="_blank">canopy height model tutorial</a> can be generalized to 
 other data products.
 
-## Setup
+<div id="ds-objectives" markdown="1">
 
-We'll need several R packages in this tutorial. Install the packages and 
-load the libraries for each:
+## Learning Objectives 
+After completing this tutorial you will be able to: 
 
-``` {r install, eval=F}
+* access NEON spatial data from the website and through data downloaded with the neonUtilities package. 
+* create a simple map with NEON domains and field site locations. 
+* access and plot specific sampling locations for TOS data products. 
+* access and use sensor location data. 
+
+## Things You’ll Need To Complete This Tutorial
+
+### R Programming Language
+You will need a current version of R to complete this tutorial. We also recommend 
+the RStudio IDE to work with R. 
+
+</div>
+
+
+## Setup R Environment
+
+We'll need several R packages in this tutorial. Install the packages, if not 
+already installed, and load the libraries for each. 
+
+```{r install, eval=F}
 
 # run once to get the package, and re-run if you need to get updates
-install.packages("sp")
-install.packages("rgdal")
-install.packages("rgeos")
-install.packages("ggplot2")
-install.packages("ggthemes")
-install.packages("neonUtilities")
-install.packages("devtools")
-devtools::install_github("NEONScience/NEON-geolocation/geoNEON")
+install.packages("sp")  # working with spatial data
+install.packages("rgdal")  # working with spatial data
+install.packages("broom")  # tidy up data
+install.packages("ggplot2")  # plotting
+install.packages("neonUtilities")  # work with NEON data
+install.packages("devtools")  # to use the install_github() function
+devtools::install_github("NEONScience/NEON-geolocation/geoNEON")  # work with NEON spatial data
 
 ```
 
-``` {r libraries, results="hide"}
+
+```{r libraries, results="hide"}
 
 # run every time you start a script
 library(sp)
 library(rgdal)
-library(rgeos)
+library(broom)
 library(ggplot2)
-library(ggthemes)
 library(neonUtilities)
 library(geoNEON)
 
 options(stringsAsFactors=F)
 
+# set working directory to ensure R can find the file we wish to import and where
+# we want to save our files. 
+
+wd <- "~/Documents/data/" # This will depend on your local environment
+setwd(wd)
+
 ```
 
-## Spatial data files
+## Source 1: NEON spatial data files
```

Comment:
Is this confusing?  In order to have an H2 header for each of the sections where we do different things to allow for easier in page navigation, I made these "super" H2 headers by calling out the data sources.

Reply:
Looks good to me!